database/migrations/postgres/1722263909995805_deployments_cutoff.up.sql

```diff
@@ -0,0 +1,8 @@
+CREATE TABLE IF NOT EXISTS queued_deployments
+(
+    id SERIAL PRIMARY KEY,
```

Comment:
please make everything `NOT NULL`

Reply:
Done for all except the `processed_at` column which will only be written once the service is deployed.